.github/actions/set-env-variables/action.yml

```diff
@@ -7,3 +7,5 @@ runs:
       run: |
         echo "QUAY_ORGANIZATION=cilium" >> $GITHUB_ENV
         echo "QUAY_ORGANIZATION_DEV=cilium" >> $GITHUB_ENV
+        # no prod yet
```

Comment:
nit/nonblocking: I don't quite get this comment

Reply:
There's no `non-dev`. Ie: There isn't a `QUAY_CHARTS_ORGANIZATION` yet, because we're only using this for dev builds (currently), not release builds. Compare this to images where we have both `QUAY_ORGANIZATION` and `QUAY_ORGANIZATION_DEV`. 

The comment could probably be "no prod charts quay org yet".

Reply:
ah right, makes sense.